force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -176,7 +207,8 @@ public with sharing class VariablesDataTypesOperators {
      * @return The average of the three numbers.
      */
     public static Double averageOfThreeNumbers(Integer a, Integer b, Integer c) {
-        return null; // Replace null with the variable you used to store the result
+        
+        return ((a + b + c)/3); // Replace null with the variable you used to store the result
```

Comment:
This works, just consider spacing standards for readability:

return ((a + b + c) / 3);